Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_executed_multiple_LDAP_enumeration_queries_Test.yml

```diff
@@ -1643,18 +1670,23 @@ tasks:
     skipunavailable: false
     task:
       brand: ''
-      description: Retrieves results from the "quser" command for the user - which
-        can be used to tell if the user is currently logged in.
-      id: f369276b-2db7-4648-8fed-32516d14d725
+      description: Retrieves results from the "quser" command for the user - which can be used to tell if the user is currently logged in.
```

Comment:
Retrieves the output of the "quser" command for the user, which can indicate whether the user is currently logged in.